### PR TITLE
Add image letter style

### PR DESCRIPTION
**Pull request by whDongRui**

### CODE REVIEW

File: src/components/AppImages/index.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { translate } from 'react-i18next';
 import { Image } from 'components/Base';
 
 import styles from './index.scss';
-import { inject } from "mobx-react/index";
+import { inject } from 'mobx-react/index';
```
Comment (iwisunny) [inline src/components/AppImages/index.jsx]: `import {inject} from 'mobx-react'`

File: src/components/AppImages/index.jsx [hunk 2]
```diff
@@ -25,22 +25,21 @@ export default class AppImages extends Component {
...
-                <Image src={icon || nonIcon} />
-              </Link>
-            ))}
+            <Link
+              className={styles.image}
+              key={app_id}
+              to={user.isDev ? `/dashboard/app/${app_id}` : `/store/${app_id}`}
+              title={name}
+            >
+              <Image src={icon} iconLetter={name} iconSize={24} className={styles.img} />
```
Comment (iwisunny) [inline src/components/AppImages/index.jsx]: iconSize is no need, default is 24px

File: src/components/Base/Image/index.jsx [hunk 3]
```diff
@@ -39,21 +41,33 @@ export default class Image extends React.Component {
...
     const { failed } = this.state;
 
     if (failed) {
       const nonIcon = '/none.svg';
-      const sizeStyle = {
+      const style = {
         width: iconSize + 'px',
         height: iconSize + 'px'
       };
+      const letter = iconLetter.substr(0, 1).toLocaleUpperCase();
```
Comment (iwisunny) [inline src/components/Base/Image/index.jsx]: refactor as 
```
iconLetter[0].toLocaleUpperCase()
```

Comment (whDongRui) [reply src/components/Base/Image/index.jsx]: ok

File: src/components/AppImages/index.jsx (hunk 2 above)
Comment (whDongRui) [reply src/components/AppImages/index.jsx]: ok

File: src/components/AppImages/index.jsx (hunk 1 above)
Comment (whDongRui) [reply src/components/AppImages/index.jsx]: ok

File: src/components/Base/Image/index.jsx (hunk 3 above)
Comment (whDongRui) [reply src/components/Base/Image/index.jsx]: if iconLetter is '', iconLetter[0] is undefined, iconLetter.substr(0, 1) is ''.

**Pull request merged**